src/App.tsx

```diff
@@ -2,108 +2,38 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+
+function getUserByID(userId: number): User | null {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+}
+
+function getCommentsByID(postId: number): Comment[] | null {
+  const filtered = [...commentsFromServer].filter(
+    (comment) => comment.postId === postId,
+  );
+
+  return filtered.length > 0 ? filtered : null;
```

Comment:
Why you should return null for it?
Just return a filtered, bc it would be empty array